tests/manage/mcg/test_object_integrity.py

```diff
@@ -63,23 +65,23 @@ def test_check_object_integrity(self, mcg_obj, awscli_pod, bucket_factory):
         argvalues=[
             pytest.param(
                 *[1, 'large'],
-                marks=[pytest.mark.polarion_id("OCS-1944"), tier2]
+                marks=[pytest.mark.polarion_id("OCS-1944"), tier2, FILESIZE_SKIP]
             ),
             pytest.param(
                 *[100, 'large'],
-                marks=[pytest.mark.polarion_id("OCS-1946"), tier3]
+                marks=[pytest.mark.polarion_id("OCS-1946"), tier3, FILESIZE_SKIP]
             ),
             pytest.param(
                 *[1, 'small'],
                 marks=[pytest.mark.polarion_id("OCS-1950"), tier2]
             ),
             pytest.param(
                 *[1000, 'small'],
-                marks=[pytest.mark.polarion_id("OCS-1951"), tier3]
+                marks=[pytest.mark.polarion_id("OCS-1951"), tier3, RUNTIME_SKIP]
             ),
             pytest.param(
                 *[100, 'large_small'],
-                marks=[pytest.mark.polarion_id("OCS-1952"), tier3]
+                marks=[pytest.mark.polarion_id("OCS-1952"), tier3, FILESIZE_SKIP]
```

Comment:
Does this mean that all those tests will be skipped?

Reply:
Those tests will be skipped. This is not really a regression, because the filesize is impractically big. The solution is to change the amount of small files to a few tens of thousands, and have one "big" file (1-10GB, not 1.6TB like now).